### PR TITLE
Use the correct install tool or source depending on how napari was installed

### PR DESCRIPTION
**Pull request by goanpeca**
Adds checks to auto select the correct tool on the dropdown depending on what is available

### CODE REVIEW

File: napari_plugin_manager/qt_plugin_dialog.py [hunk 1]
```diff
@@ -101,10 +102,10 @@ def __init__(
 
         self.plugin_name.setText(name)
 
-        if len(versions_pypi) > 0:
-            self._populate_version_dropdown(PYPI)
-        else:
+        if len(versions_pypi) == 0 and len(versions_conda) > 0:
             self._populate_version_dropdown(CONDA)
+        elif len(versions_pypi) > 0 and len(versions_pypi) == 0:
```
Comment (jaimergp) [inline napari_plugin_manager/qt_plugin_dialog.py:107]: What if we have versions in both?

Comment (goanpeca) [reply napari_plugin_manager/qt_plugin_dialog.py]: Fixed!

**Comment by dalthviz** [comment]
Hi there, gave this a manual check and seems like things are working as expected :+1: Also, just in case, a couple of GIFs showing how things are looking:

* PyPI based installation:

install_pypi

* Conda based installation:

install_conda

**Review by dalthviz — APPROVED** [review]

**Comment by jaimergp** [comment]
Oh, yes, definitely! Well spotted!

**Pull request merged**